bycycle/burst/utils.py

```diff
@@ -20,6 +20,13 @@ def check_min_burst_cycles(is_burst, min_n_cycles=3):
     -------
     is_burst : 1d array
         Updated burst array.
+
+    Examples
+    --------
+    Removed bursts with less than 3 consectutive cycles.
```

Comment:
tense nitpick, but I think it should be:
`Remove bursts with less than 3 consectutive cycles:`

Present tense, and with trailing colon, like other docstrings.